pkg/utils/utils.go

```diff
@@ -295,3 +300,35 @@ func getPlatform() string {
 	}
 	return PlatformK8s
 }
+
+// GetOCPIngressDomain returns the OpenShift Ingress domain
+func GetOCPIngressDomain(cl client.Client) (string, error) {
```

Comment:
Can we move it to controller level (it is called from there I think) to not to make REST calls from utils?

Reply:
Updated in [`d127f5f` (#892)](https://github.com/redhat-developer/rhdh-operator/pull/892/commits/d127f5f315288f14807d47545000dabd0f6b19cf)